smartsim/_core/entrypoints/dragon.py

```diff
@@ -138,6 +173,10 @@ def main(args: argparse.Namespace) -> int:
 
     if args.launching_address:
         zmq_context = zmq.Context()
+        zmq_context.setsockopt(zmq.SNDTIMEO, value=-1)
+        zmq_context.setsockopt(zmq.RCVTIMEO, value=-1)
+        zmq_context.setsockopt(zmq.REQ_CORRELATE, 1)
+        zmq_context.setsockopt(zmq.REQ_RELAXED, 1)
```

Comment:
We set these four context options multiple times. Can we add a function instead of repeating the lines?

Reply:
It is now set once